src/resources/filters/quarto-post/render-asciidoc.lua

```diff
@@ -60,16 +60,26 @@ function render_asciidoc()
       -- render the callout contents
       local admonitionContents = pandoc.write(pandoc.Pandoc(el.content), "asciidoc")
```

Comment:
I believe you can remove this now right ?

Reply:
Oh yeah, thanks. I wish the Lua mode were more aggressive in showing unused variables.

Reply:
Fixed on `main`.